moveit_core/planning_request_adapter/src/planning_request_adapter.cpp

```diff
@@ -157,7 +163,7 @@ bool PlanningRequestAdapterChain::adaptAndPlan(const planning_interface::Planner
         added_path_index.push_back(added_index);
       }
     std::sort(added_path_index.begin(), added_path_index.end());
-    return result;
+    return res.error_code_.val == moveit_msgs::msg::MoveItErrorCodes::SUCCESS;
```

Comment:
I do really like the direction of this change. Even better would be to have all these functions return the error type instead of the bool, but I know that is probably an even larger change.